### DISCUSSION TITLE
David Farrell : Leaving the Bash-It Team

### DISCUSSION THREAD

**Discussion by davidpfarrell** [Bash-it/bash-it#2255] [General]
Hi Team!

I tried for 30 minutes to find a way to have a private maintainer-only discussion and my search-fu failed me, so posting here.

I've decided to leave the Bash-It teams i'm a member of.

I had to move onto zsh for my work laptop and really haven't spent any time in Bash-It since.   Not that zsh is better or anything, just that its what I'm using now.

I always loved the idea of Bash-it and wish the project much success!

Sincerely,
-DF

**Comment by cornfeedhobo** [comment]
@davidpfarrell I totally understand. Thank you for your hard work and contributions. I'll be sure to ping you if I ever get my other multi-terminal project into a releasable state :)
